tsconfig.json

```diff
@@ -12,7 +12,7 @@
     // "jsx": "preserve",                           /* Specify JSX code generation: 'preserve', 'react-native', 'react', 'react-jsx' or 'react-jsxdev'. */
     "declaration": true /* Generates corresponding '.d.ts' file. */,
     // "declarationMap": true,                      /* Generates a sourcemap for each corresponding '.d.ts' file. */
-    // "sourceMap": true,                           /* Generates corresponding '.map' file. */
+    "sourceMap": true /* Generates corresponding '.map' file. */,
```

Comment:
Do we need this?

Reply:
@shubham1172 I enabled this so I could debug the code. I'm not a bit typescript person though, so if you think it's better not to have it in a release, that's totally ok for me, I'll just enable it for my local development.

Reply:
Since we use the same tsconfig for dev and prod, the production build size will also increase. We could add support to conditionally generate sourceMap for development only, but for now we should turn this off.

Reply:
Has this been resolved meanwhile?